### ISSUE TITLE
Gif width and hight not working

### ISSUE THREAD

**Issue by mehmetbalbay** [adhishlal/GifLoader#2]
<lal.adhish.gifprogressbar.GifView
            android:id="@+id/progressBar"
            **android:layout_width="@dimen/_50dp"**
            android:layout_centerInParent="true"
            android:layout_margin="@dimen/_5sdp"
            **android:layout_height="@dimen/_50dp"**
            android:layout_gravity="center" />

**Comment by rlogical-techsoft** [comment]
Correct. This is not working. I have tried by providing dynamic height and width as well but still it's not working.
